content/en/docs/components/pipelines/operator-guides/configure-object-store.md

```diff
@@ -0,0 +1,404 @@
+
+## KFP API Server
+
+The KFP API Server uses the object store to store the Pipeline Intermediate Representation (IR).
+
+The list below describe the type of Object Store configurations supported today for API Server. Static credentials here
+refers to long term credentials provided by Object Store providers. 
+
+For [AWS Static Credentials] and other S3 compliant object storage, this consists of an Access Key ID and Secret Access Key ID embedded into the run time environment or, passed as secure parameters to some API. In Google Cloud Storage, this refers to a JSON containing the [GCS APP Credentials].
+
+### API Server Supported providers
+
+| Provider                                     | Supported |
+|----------------------------------------------|-----------|
+| Minio with Static Credentials                | Yes       |
+| AWS S3 with Static Credentials               | Yes       |
+| AWS S3 with IRSA                             | Yes       |
+| S3-Compliant Storage with Static Credentials | Yes       |
+| Google Cloud Storage with Static Credentials | No        |        
+| Google Cloud Storage with App Credentials    | No        |  
+
+### API Server Object Store Configuration
+
+To configure the object store used by the KFP API Server, the configuration depends on whether you are using static credentials, or AWS S3 with IRSA.
+
+**Static credentials**
+
+To configure an AWS S3 bucket with static credentials you will need to add the following environment variables to your KFP API Server deployment:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ml-pipeline
+  namespace: kubeflow
+spec:
+  ...
+  template:
+    ...
+    spec:
+      containers:
+      - name: ml-pipeline-api-server
+        serviceAccountName: "ml-pipeline"
+        env:
+          ...
+          - name: OBJECTSTORECONFIG_HOST
+            value: "your-bucket" # e.g. s3.amazonaws.com
+          - name: OBJECTSTORECONFIG_PORT
+            value: "port" # e.g. 443   
+          - name: OBJECTSTORECONFIG_REGION
+            value: "region" # e.g. us-east-1
+            # true if object store is on a secure connection 
+          - name: OBJECTSTORECONFIG_SECURE
+            value: "true"        
+            # These env vars reference the values from a Kubernetes secret
+            # this requires deploying the secret ahead of time, and filling out the
+            # following values accordingly.
+          - name: OBJECTSTORECONFIG_ACCESSKEY
+            valueFrom:
+              secretKeyRef:
+                key: "some-key-1"
+                name: "secret-name"
+          - name: OBJECTSTORECONFIG_SECRETACCESSKEY
+            valueFrom:
+              secretKeyRef:
+                key: "some-key-2"
+                name: "secret-name"
+```
+
+**AWS IRSA (IAM Roles for Service Accounts)**
```

Comment:
Take a look at my previous comment. The full name should be referenced the first time you mention it